.github/CODEOWNERS

```diff
@@ -0,0 +1,81 @@
+azeeed/* @roasbeef @cfromknecht
+
+autopilot/* @halseth @roasbeef
+
+brontide/* @roasbeef @cfromknecht
+
+buffer/* @cfromknecht
+
+build/* @cfromknecht
+
+chainntnfs/* @halseth @wpaulino
+
+chanbackup/* @roasbeef
+
+channeldb/channel.go @roasbeef
```

Comment:
do we need a `channeldb` catch all for files other than those listed?

Reply:
I don't think so? The post mentions [a concept of `global owners`](https://help.github.com/en/articles/about-code-owners), but it's unclear if that's a default, or if we'd need to make it ourselves.

Reply:
it looks like we need to define them first:
```
# These owners will be the default owners for everything in
# the repo. Unless a later match takes precedence,
# @global-owner1 and @global-owner2 will be requested for
# review when someone opens a pull request.
*       @global-owner1 @global-owner2
```
so then if something matches after it overrides the global owners

Reply:
Added